### ISSUE TITLE
“Pointer Input” not well defined in Glossary

### ISSUE THREAD

**Issue by yatil** [w3c/wcag#1762]
The [Pointer Input Glossary item](https://www.w3.org/TR/WCAG21/#dfn-pointer-inputs) reads: 

 > **pointer input**
> input device that can target a specific coordinate (or set of coordinates) on a screen, such as a mouse, pen, or touch contact
> 
> See also Pointer Events pointer definition [pointerevents].

The link points to the [definition of pointer](https://www.w3.org/TR/pointerevents/#dfn-pointer) which reads:

> **pointer**
> A hardware agnostic representation of input devices that can target a specific coordinate (or set of coordinates) on a screen, such as a mouse, pen, or touch contact.

The WCAG definition is used in [SC 2.5.5 Target Size](https://www.w3.org/TR/WCAG21/#target-size): 

> The size of the target for pointer inputs is at least 44 by 44 CSS pixels except when:

I don't think it makes sense to have pointer defined as the “hardware agnostic representation of input devices” and then define “pointer input” as those “input devices”. 

I think what is meant in SC 2.5.5 is the actual element that is targeted. 

The glossary should change to something along these lines:

 > **pointer input**
> input mechanism for a [pointer](https://www.w3.org/TR/pointerevents/#dfn-pointer) [pointerevents], for example a link or button which is supposed to be clicked, touched, or otherwise activated with a pointing device

(I’m unsure if I have understood this correctly, but if the device is meant, why don’t we say “The size of the target for pointers [instead of pointer inputs] is at least 44 by 44 CSS pixels except when:” in SC 2.5.5? That is an unnecessary re-definition in that case.)

Related: #749/#809

**Comment by patrickhlauke** [comment]
agree, have not been happy with the bastardisation/redefinition of "pointer input" to then talk directly about the device/hardware. #809 went through various attempts to square that circle without needing to redefine it. I might make another stab at that PR, perhaps a bit wordy...

> input/action carried out with a pointer device

or something along those lines (as it's not an input *mechanism* as per your suggestion, but more the actual input event/result itself that was done with the hardware)

**Comment by patrickhlauke** [comment]
ok, had another stab at improving things in https://github.com/w3c/wcag/pull/809 ... maybe at some point this almost 2 year old PR may get a look in ;)

**Comment by yatil** [comment]
Thanks @patrickhlauke!

**Comment by yatil** [comment]
So, still no resolution here, and it looks like the bad definition is also going to make its way into WCAG 2.2. Is there anything we can do to fix it before 2.2 gets published?

**Comment by yatil** [comment]
Fixed in
- https://github.com/w3c/wcag/pull/809

I can't believe that it happened. Thanks @patrickhlauke!
